package.json

```diff
@@ -18,14 +20,15 @@
   },
   "devDependencies": {
     "@cypress/react18": "^2.0.1",
-    "@mate-academy/scripts": "^1.8.5",
+    "@mate-academy/scripts": "^1.9.12",
     "@mate-academy/students-ts-config": "*",
     "@mate-academy/stylelint-config": "*",
     "@types/node": "^20.14.10",
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
     "@types/react-redux": "^7.1.33",
     "@types/react-transition-group": "^4.4.10",
+    "@typescript-eslint/eslint-plugin": "^8.8.0",
```

Comment:
Do you really need it? 
![image](https://github.com/user-attachments/assets/76ad56e8-06d2-432b-ae5c-05935f03ce42)

I guess locally you did `npm I --legacy-per-deps` to fix it. But not sure you need this version. If yes - you can try to change "@typescript-eslint/parser": "^7.16.0" version to the same (see error)

Reply:
My solution was to clone the repo into a new folder and reinstall the dependencies. Thanks!